Dokan/Dokan/Scenes/ProductDetails/Views/SliderCollectionCell/SliderCollectionViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SliderCollectionViewCell.swift
+//  Dokan
+//
+//  Created by heba isaa on 19/07/2022.
+//
+
+import UIKit
+import UIDokan
+
+class SliderCollectionViewCell: UICollectionViewCell, IdentifiableView {
+
+    // MARK: - Outlet
+
+    @IBOutlet private weak var productImageView: UIImageView!
+    @IBOutlet private weak var noOfCurrentPictureLabel: UILabel!
+    @IBOutlet private weak var noOfPicturesLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+    }
+
+ // MARK: - Configurations
+
+    func configureCellData(SliderCollectionViewCellModel: sliderViewModel) {
```

Comment:
Shorter descriptive names are preferred. 
```suggestion
    func configureCell(viewModel: sliderViewModel) {
```